src/keys.ts

```diff
@@ -28,15 +28,18 @@ export const EXTENDED_PUBLIC_KEY_VERSIONS = {
   vpub: "045f1cf6",
   Upub: "024289ef",
   Vpub: "02575483",
-};
+} as const;
+
+type KeyPrefixes = keyof typeof EXTENDED_PUBLIC_KEY_VERSIONS;
```

Comment:
Is this only required for the next line? Or could this be used in `validatePrefix` ?

Reply:
good call!